aea/configurations/loader.py

```diff
@@ -252,34 +252,14 @@ def from_configuration_type(
         configuration_type = PackageType(configuration_type)
         return ConfigLoaders.from_package_type(configuration_type)
 
-    def _validate(self, json_data: Dict) -> None:
```

Comment:
No longer used?

Reply:
yeah, the only reason was to check the 'type' field. Now the check is incorporated in the JSON schema validation.